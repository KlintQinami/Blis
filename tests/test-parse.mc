int main()
{
  int a;
  printb(string_to_integer("42", a));
  printi(a);

  float b;
  printb(string_to_float("42.", b));
  printf(b);

  printb(string_to_float("-0.1", b));
  printf(b);

  return 0;
}

