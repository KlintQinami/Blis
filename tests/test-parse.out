1
42
1
42.000000
1
-0.100000
